### ISSUE TITLE
Crash when no bias or batch norm beta

### ISSUE THREAD

**Issue by benjaminkreis** [hls-fpga-machine-learning/hls4ml#126]
@sergojin has some models with the dense layer bias or batch norm beta turned off.  hls4ml crashes trying to find them in the h5 file.

I've started a branch with the models (keras-config.NoBias.yml and keras-config.NoBeta.yml) and a potential fix in keras-to-hls.py.  If it doesn't find the bias or batch norm beta, it uses an array of zeros instead.  I've only thought about it so far in the context of dense layers and still need to think about what changes for other layer types (e.g. cnn).  

the branch is here: https://github.com/hls-fpga-machine-learning/hls4ml/tree/offset_bug 

Note these models will not synthesize due to #104 but the extraction from the h5 and building of batch norm or dense layers can still be tested with them.
